great_expectations/datasource/fluent/interfaces.py

```diff
@@ -350,6 +351,8 @@ class Datasource(
 
     # class attrs
     asset_types: ClassVar[Sequence[Type[DataAsset]]] = []
+    # Not all Datasources require a DataConnector
+    data_connector_type: ClassVar[Optional[Type[DataConnector]]] = None
```

Comment:
@Kilo59 If this addition is toward including `DataConnector` in Fluent configuration, then we are not supposed to move in that direction (because in Fluent, `DataConnector` is an implementational utility, not a core concept).  If this is for some other purpose, then please let me know and I will study more carefully.  Thanks.

Reply:
No, it's about ensuring we don't need `DataConnector` in the configuration.

Reply:
For my edification: Sounds like if a property is `None`, it will not end up in Configuration?  Or if this is incorrect, how does one specify a property "to be part of configuration" and "to not be part of configuration"?  Thanks.

Reply:
We check this class attribute to know whether or not the Datasource and data assets need to utilize a data_connector. If it does, we need to build that specific `DataConnector` with this attribute.